java/code/src/com/suse/manager/utils/SaltUtils.java

```diff
@@ -231,6 +231,8 @@ public boolean shouldRefreshPackageList(String function,
     }
 
     /**
+     * Change this file to trigger workflow.
+     *
```

Comment:
Of course this commit will be dropped before merging, right?

Reply:
Yes, I will drop it before merging, it is just here to test the workflow.